src/API/Query.php

```diff
@@ -916,6 +956,9 @@ protected static function convertBiasing($biasing)
         $convertedBiasing->setAugmentBiases($biasing->isAugmentBiases());
         $hasData = true;
       }
+      if ($biasing->getNumericBoosts() != array()) {
```

Comment:
@grrilla You should instead use PHP's `empty()` function here, i.e;
```
if (! empty($biasing->getNumericBoosts())) {...
```

Reply:
the empty() function allegedly only takes vars, not returned values from functions. That code you've written would only work with PHP 5.5 or later, so this comparison structure is used throughout the API. 

http://php.net/manual/en/function.empty.php

Reply:
Oh wow I didn't even realize this, good catch!